src/VisualStudio/Setup.Dependencies/Roslyn.VisualStudio.Setup.Dependencies.csproj

```diff
@@ -35,6 +35,7 @@
     <NuGetPackageToIncludeInVsix Include="System.Runtime.CompilerServices.Unsafe" PkgDefEntry="BindingRedirect" />
     <NuGetPackageToIncludeInVsix Include="System.Text.Encoding.CodePages" PkgDefEntry="BindingRedirect" />
     <NuGetPackageToIncludeInVsix Include="System.Threading.Tasks.Extensions" PkgDefEntry="BindingRedirect" />
+    <NuGetPackageToIncludeInVsix Include="System.IO.Pipelines" PkgDefEntry="BindingRedirect" />
```

Comment:
What needed this?

Reply:
This was required because the version of this in the integration test machine was different from that of roslyn master.  However now we're on preview 4 I might be able to remove this.  Let me check.

Reply:
I guess I was surprised because I didn't see any other packages being updated and I don't think you were using it directly, so it wasn't clear to me what was depending on it.